### DISCUSSION TITLE
Need Assistance Or Advice About Unknown Devices Discovered In large quantity

### DISCUSSION THREAD

**Discussion by MarkiBamboo** [seemoo-lab/AirGuard#191] [General]
I don't know who to contact about advice or assistance in dealing with the picture attached. There are 9 unknown Bluetooth devices that are surrounding my room. I need to have a private way to communicate because I'm not comfortable having this openly discussed. I can not use your app because I have an iPhone. I used Find My Bluetooth Device. I reached out to Mr. Heinrich but his automatic reply says he is unable to communicate until the 16th. It seemed important to have some contact that might understand this. I have more information to share as well.
![IMG_0242](https://github.com/user-attachments/assets/9701480f-3a42-4096-bcaf-2622f427104b)
![IMG_0242](https://github.com/user-attachments/assets/a4390cfd-d6a4-4145-8c89-151cc0713859)

**Comment by rec0de** [comment]
Hi!

There are lots of Bluetooth devices out there being used for all sorts of purposes - so this is not an immediate cause for concern.

If you want to check for trackers using your iPhone, you can use the iOS version of AirGuard: https://github.com/seemoo-lab/AirGuard-iOS/

**Comment by MarkiBamboo** [comment]
I'm not sure if you actually realize that your app only searches for
Android device Bluetooth frequencies. Your app simply won't work for Apple
frequencies. This is the current screenshot I just took at 7 AM Eastern
time. I told you the they are unknown devices. Nothing is unknown unless
Apple doesn't have the frequency it's sending. The entire concentration of
all the red dots should be shocking to you. I have been so scared because
I'm not exactly sure has put them there. I have only 4 Bluetooth devices
and I explained to you that my own are excluded from the search which means
I obviously understand what a Bluetooth device is. Is that as descriptive
as I need to be. Right now there are probably so many more that it can't
even get the ones the app sees in the other rooms when I search there. This
is maybe one of the highest concentrations ever on one single person while
I'm in my bedroom. They must be in at least the ceiling and the walls and
doors and there are actually outside the walls of my room. I contacted you
because Alexander Heinrich's out of office message told me to contact his
colleagues. He may be the foremost authority on stalking and abuse because
he put so many types in each question that he obviously couldn't fit them
on the page. I guarantee if he hears that one stalking victim were ignored
just because I'm a man he might pull the app forever. If he gave his
personal email rely that I am attaching a screenshot of that he may have
already seen my message immediately and may be waiting until I gave him to
confirm their locations. I even gave him my mothers name, firmer location
in _[personal information removed]_ and then I gave him my mother's birth city of _[personal information removed]_ which is
an East German capital for this kind of abuse. I have cried so much from
the sound torture alone that doesn't even include the verbal, mental, and
emotional abuse which is far more extensive. This is obviously about
stalking which is his entire purpose of the app. I mean do you think I drew
circles on paper before I sent a screenshotwith the obvious face of an
IPhone.  You don't understand that it's from my family mostly my sister and
mother for sure and I know others. Do you know how much courage it takes to
report your mother as a stalker? My sister _[personal information removed]_ is far
worse. I finally found an app that could locate these and was so happy that
I could have then understand that my life is literally threatened beyond
belief as soon as I walk into the door of the last safe space I have left.
This "game" was one of the worst tortures they took from children's rooms
to the gas chamber.  I'm reaching out to you for emergency help. How much
longer can I be expected to endure this? I just gave you specific
information that implies so much more which I have pictured and recorded on
my phone. If you don't at least look at them then you could be condoning
behavior that can lead to the then murdering me. I don't need to explain to
you that I'm not crazy. If you just look at my words and the complexity of
my descriptions, you'll understand that nobody on any drug especially
crystal. Meth could ever write this clearly. My mother grew up on
Pervectin.  She's insane but 82. Should I just walk out and scream Bye? You
are actually the first person I've even been able to tell about this. It's
been happening for so many years. I know I may have not even understood the
things I said to me or meant to hurt me. I just need some help and can you
please Get Alexander Heinrich on the phone or please communicate this to
him because he might be waiting for you. Finally listen to him to save
somebody's life that happened to have each German Nazi dad as her father.
Ask him how they treated their kids and gave them perfection which is a
crystal meth medication because it was over-the-counter then.

_[personal information removed]_
